client/state/data-layer/wpcom/reader/teams/index.js

```diff
@@ -4,20 +4,26 @@
 import { READER_TEAMS_REQUEST, READER_TEAMS_RECEIVE } from 'state/action-types';
 import wpcom from 'lib/wp';
 
-export function handleTeamsRequest( state, action, next ) {
-	wpcom.req.get( '/read/teams', { apiVersion: '1.2' } )
+export function handleTeamsRequest( store, action, next ) {
+	return wpcom.req.get( '/read/teams', { apiVersion: '1.2' } )
```

Comment:
if the URL for the endpoint is `/read/teams` then this file should match that, even though `reader` looks nicer. the goal is to make it easy to find what code relates to which endpoints and as the gateway between the two worlds: Calypso and the WordPress.com API we really don't want to be introducing confusion here.